src/VirtoCommerce.Platform.Modules/Local/LocalStorageDistributedLockResource.cs

```diff
@@ -0,0 +1,41 @@
+using System;
+using System.IO;
+using VirtoCommerce.Platform.Core.DistributedLock;
+using VirtoCommerce.Platform.Core.Exceptions;
+
+namespace VirtoCommerce.Platform.Modules
+{
+    /// <summary>
+    /// Distributed lock for local storage.
+    /// Creates some file with Guid to mark the storage. Next, this Guid used as a resource id for distributed lock.    
```

Comment:
/// Creates a file with Guid to mark the storage. Next, this Guid is used as a resource id for distributed lock.